### PR TITLE
Bug/lab time only once

### PR DESCRIPTION
**Pull request by DominikGuzei**
fixes issue that lab time was only recorded the first time. Also removes a test case that made no sense for me.

### CODE REVIEW

**Comment by tombu** [comment]
This was not a bug. The tracked time was the race time. The actual bug was that you could drive although you already crossed the finish line. Until now our target was to be able to drive one lap (according to the user stories). :wink:

File: app/assets/javascripts/slotcars/play/controllers/game_controller.js.coffee [hunk 1]
```diff
@@ -46,10 +45,11 @@ slotcars.play.controllers.GameController = Ember.Object.extend
     @gameLoopController.start => @update()
 
   finish: ->
-    (jQuery @car).off 'crossFinishLine'
     @endTime = new Date().getTime()
     @set 'raceTime', @endTime - @startTime
     @car.reset()
+    @_resetCarPosition()
+    @startTime = new Date().getTime()
```
Comment (tombu) [inline app/assets/javascripts/slotcars/play/controllers/game_controller.js.coffee:52]: This seems to me like a quick fix. Setting the start time is not what in the finish method should do.

File: app/assets/javascripts/slotcars/play/controllers/game_controller.js.coffee [hunk 2]
```diff
@@ -46,10 +45,11 @@ slotcars.play.controllers.GameController = Ember.Object.extend
     @gameLoopController.start => @update()
 
   finish: ->
-    (jQuery @car).off 'crossFinishLine'
```
Comment (tombu) [inline app/assets/javascripts/slotcars/play/controllers/game_controller.js.coffee:49]: Exactly this is what should happen if you finish a race!

**Comment by tombu** [comment]
Like already mentioned in one of the comments: 'just a bugfix'. I don´t like this solution. The lap time should be a feature and no bugfix. Therefore we should at least create a new method and save the lap times anywhere.

**Comment by DominikGuzei** [comment]
it definitely was a quick fix and the whole lab / race time logic should be clearer to understand. You're totally right in that this is actually its own feature and the bugfix would be to stop updating the car position after the race finished.

**Comment by tombu** [comment]
So should it be merged? Do we want it? In my opinion we should leave it and close the pull request without merging.

**Comment by DominikGuzei** [comment]
Yeah, since it was not really the right fix I'll close it! :-)

**Pull request closed without merging**